idaes/models_extra/column_models/tests/test_plate_heat_exchanger.py

```diff
@@ -211,10 +211,10 @@ def test_solution(self, phe):
 
         assert pytest.approx(244327, rel=1e-5) == value(phe.fs.unit.heat_duty[0])
 
-        assert pytest.approx(365.288, rel=1e-5) == value(
+        assert pytest.approx(345.612, rel=1e-5) == value(
```

Comment:
These are some significant changes in the model results - have you confirmed that you have verified these changes are correct?